data/skipper-entrypoint.sh

```diff
@@ -1,10 +1,17 @@
 #!/usr/bin/env bash
 
+SKIP_HOME_DIR_PARAM=""
+# if home directory for user already exists, then useradd should not try to create it,
+# but we'll need to make sure that the user is owner of it
```

Comment:
Update the comment: "If home directory already exists, useradd shouldn't create it"
Consider chaning "-M" to: "--no-create-home"